CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Users can now export `SimulationData` to MATLAB `.mat` files with the `to_mat_file` method.
 - `ModeSolver` methods to plot the mode plane simulation components, including `.plot()`, `.plot_eps()`, `.plot_structures_eps()`, `.plot_grid()`, and `.plot_pml()`.
 - Support for differentiation with respect to monitor attributes that require interpolation, such as flux and intensity.
+- Broadband adjoint support in autograd for adjoint sources with the same spatial dependence.
+- Support for automatic differentiation with respect to `.eps_inf` and `.poles` contained in dispersive mediums `td.PoleResidue` and `td.CustomPoleResidue`.
```

Comment:
Dispersive probably shouldn't be in here?

Reply:
Yea, it turned out that dispersive was in the wrong place, I'll fix when I rebase.